index.js

```diff
@@ -1,21 +1,41 @@
 'use strict';
 
 const config = require('./config');
-const amqp = require('amqp-wrapper')(config.amqp);
-const knex = require('knex')(config.db.knex);
 const debug = require('debug')('amqp-log-to-pg');
 
-knex.schema.createTableIfNotExists(config.db.tableName, function (table) {
-  table.increments();
-  table.jsonb('data');
-  table.timestamps(true, true);
-}).then(function () {
-  amqp.connect().then(function () {
-    amqp.consume(onMessage);
-  }).catch(console.error);
-});
+let hasMeta;
+function migrate (knex) {
```

Comment:
any reason why you have chosen to roll your own migrate process rather than using the usual migrations dir with knex migrate?